e2e/e2e-summary.js

```diff
@@ -0,0 +1,263 @@
+#!/usr/bin/env node --experimental-modules
+
+import Web3 from "web3"
+import ProviderEngine from "web3-provider-engine"
+import WebsocketSubprovider from "web3-provider-engine/subproviders/websocket.js"
+import TBTC from "@keep-network/tbtc.js"
+import Subproviders from "@0x/subproviders"
+import {getTBTCTokenBalance} from "./common.js";
+import program from "commander"
+import * as fs from 'fs'
+import web3Utils from "web3-utils"
+
+const { toBN } = web3Utils
+
+program 
+.option('--bitcoin-electrum-host <host>', "electrum server host", "127.0.0.1")
+.option('--bitcoin-electrum-port <port>', "electrum server port", (port) => parseInt(port, 10), 50003)
+.option('--bitcoin-network <network>', "type of the bitcoin network (\"regtest\"|\"testnet\")", "regtest")
+.option('--ethereum-node <url>', "ethereum node url", "ws://127.0.0.1:8546")
+.option('--ethereum-pk <privateKey>', "private key of ethereum account", "f95e1da038f1fd240cb0c966d8826fb5c0369407f76f34736a5c381da7ca0ecd")
+.option('--blocks-timespan <blocksTimespan>', "blocks to search back from", "500")
+.parse(process.argv)
+
+console.log("\nScript options values: ", program.opts(), "\n")
+
+let tbtc
+const blocksTimespan = parseInt(program.blocksTimespan, 10)
```

Comment:
You can add the parse function directly in the option definition. Look at `--bitcoin-electrum-port`. This way you can use `program.blocksTimespan` directly and it will be a `number`.

Reply:
yup, good call. Changed